components/cli/vendor/github.com/docker/docker/pkg/pools/pools.go

```diff
@@ -17,15 +17,16 @@ import (
 	"github.com/docker/docker/pkg/ioutils"
 )
 
+const buffer32K = 32 * 1024
```

Comment:
this change to `components/cli` was a result of a missing `vendor` update when #76 was cherry-picked to `components/engine`. should have been a separate commit in PR #76 to update the `vendor`